### PR TITLE
Regularize structure naming conventions

### PR DESCRIPTION
**Pull request by ecavallo**

### CODE REVIEW

File: Cubical/Experiments/Rng.agda [hunk 1]
```diff
@@ -4,15 +4,16 @@
 -- rewrite has been postponed.
 
 {-# OPTIONS --cubical --no-import-sorts --safe #-}
-module Cubical.Structures.Rng where
+module Cubical.Experiments.Rng where
```
Comment (mortberg) [inline Cubical/Experiments/Rng.agda:7]: Oops, this wasn't caught by Travis... Can you add this file to Experiments.Everything?

File: Cubical/Foundations/SIP.agda [hunk 2]
```diff
@@ -31,57 +31,60 @@ private
 -- a proposition. Indeed this type should correspond to the ways s and t can be identified
 -- as S-structures. This we call a standard notion of structure or SNS.
 -- We will use a different definition, but the two definitions are interchangeable.
-SNS-≡ : (S : Type ℓ₁ → Type ℓ₂) (ι : StrIso S ℓ₃) → Type (ℓ-max (ℓ-max (ℓ-suc ℓ₁) ℓ₂) ℓ₃)
-SNS-≡ {ℓ₁} S ι = ∀ {X : Type ℓ₁} (s t : S X) → ι (X , s) (X , t) (idEquiv X) ≃ (s ≡ t)
+UnivalentStr-≡ : (S : Type ℓ₁ → Type ℓ₂) (ι : StrEquiv S ℓ₃) → Type (ℓ-max (ℓ-max (ℓ-suc ℓ₁) ℓ₂) ℓ₃)
```
Comment (mortberg) [inline Cubical/Foundations/SIP.agda:34]: I think this should be called SNS as it is Martin's original notion

File: Cubical/Foundations/SIP.agda [hunk 3]
```diff
@@ -31,57 +31,60 @@ private
...
-                                   s ≡ t                                   ■
+UnivalentStr→UnivalentStr-≡ : (S : Type ℓ₁ → Type ℓ₂) (ι : StrEquiv S ℓ₃)
+  → UnivalentStr S ι → UnivalentStr-≡ S ι
+UnivalentStr→UnivalentStr-≡ S ι θ {X = X} s t =
+  ι (X , s) (X , t) (idEquiv X)           ≃⟨ θ (idEquiv X) ⟩
+  PathP (λ i → S (ua (idEquiv X) i)) s t  ≃⟨ φ ⟩
+  s ≡ t                                   ■
   where
-   φ = transportEquiv (λ j → PathP (λ i → S (uaIdEquiv {A = X} j i)) s t)
+  φ = transportEquiv (λ j → PathP (λ i → S (uaIdEquiv {A = X} j i)) s t)
```
Comment (mortberg) [inline Cubical/Foundations/SIP.agda:63]: Does this really need to be in a where? I would inline it

File: Cubical/Foundations/SIP.agda [hunk 4]
```diff
@@ -31,57 +31,60 @@ private
...
-   C s t = ι (Y , s) (Y , t) (idEquiv Y)           ≃⟨ θ s t ⟩
-           s ≡ t                                   ≃⟨ ψ ⟩
-           PathP (λ i → S (ua (idEquiv Y) i)) s t  ■
+  C : (s t : S Y) → ι (Y , s) (Y , t) (idEquiv Y) ≃ PathP (λ i → S (ua (idEquiv Y) i)) s t
+  C s t = ι (Y , s) (Y , t) (idEquiv Y)           ≃⟨ θ s t ⟩
+          s ≡ t                                   ≃⟨ ψ ⟩
+          PathP (λ i → S (ua (idEquiv Y) i)) s t  ■
     where
-     ψ = transportEquiv λ j → PathP (λ i → S (uaIdEquiv {A = Y} (~ j) i)) s t
+    ψ = transportEquiv λ j → PathP (λ i → S (uaIdEquiv {A = Y} (~ j) i)) s t
```
Comment (mortberg) [inline Cubical/Foundations/SIP.agda:79]: Inline?

File: Cubical/Structures/Axiom.agda [hunk 5]
```diff
@@ -0,0 +1,43 @@
+{-# OPTIONS --cubical --no-import-sorts --safe #-}
+module Cubical.Structures.Axiom where
```
Comment (mortberg) [inline Cubical/Structures/Axiom.agda:2]: Wouldn't this read nicer if it was `Axioms`? (I don't feel super strongly about this so feel free to ignore)

File: Cubical/Structures/Axiom.agda [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+open import Cubical.Foundations.SIP
+open import Cubical.Data.Sigma
+
+private
+  variable
+    ℓ ℓ₁ ℓ₁' ℓ₂ : Level
+
+-- Now, we want to add axioms (i.e. propositions) to our Structure S that don't affect the ι.
+-- We use a lemma due to Zesen Qian, which can now be found in Foundations.Prelude:
+-- https://github.com/riaqn/cubical/blob/hgroup/Cubical/Data/Group/Properties.agda#L83
```
Comment (mortberg) [inline Cubical/Structures/Axiom.agda:19]: Do we use this lemma?

File: Cubical/Structures/Axiom.agda [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+         → StrEquiv (AxiomStructure S axioms) ℓ₁'
+AxiomEquivStr ι axioms (X , (s , a)) (Y , (t , b)) f = ι (X , s) (Y , t) f
+
+axiomUnivalentStr : {S : Type ℓ → Type ℓ₁}
+                 (ι : (A B : Σ[ X ∈ (Type ℓ) ] (S X)) → A .fst ≃ B .fst → Type ℓ₁')
+                 {axioms : (X : Type ℓ) → S X → Type ℓ₂}
+                 (axioms-are-Props : (X : Type ℓ) (s : S X) → isProp (axioms X s))
+                 (θ : UnivalentStr S ι)
+               → UnivalentStr (AxiomStructure S axioms) (AxiomEquivStr ι axioms)
+axiomUnivalentStr {S = S} ι {axioms = axioms} axioms-are-Props θ {X , s , a} {Y , t , b} f =
```
Comment (mortberg) [inline Cubical/Structures/Axiom.agda:37]: Can you rewrite this using \simeq-reasoning? It's very opaque... But super nice to have this simpler proof! (@mzeuner )

File: Cubical/Structures/Axiom.agda (hunk 6 above)
Comment (ecavallo) [reply Cubical/Structures/Axiom.agda:19]: I'm not really sure what it's referring to.

**Comment by ecavallo** [comment]
By the way, I think we should delete `Cubical.Experiments.HInt` given that we have `Cubical.HITs.Ints.HAEquivInt`.

**Comment by mortberg** [comment]
> By the way, I think we should delete `Cubical.Experiments.HInt` given that we have `Cubical.HITs.Ints.HAEquivInt`.

Nothing in the library should use anything in the Experiments folder so I think it could be kept as it is as an open problem. The comment in the top of the file should be updated with a link to `Cubical.HITs.Ints.HAEquivInt` though.

File: Cubical/Structures/Axiom.agda (hunk 6 above)
Comment (mortberg) [reply Cubical/Structures/Axiom.agda]: I think it's outdated and should be deleted

File: Cubical/Structures/Axioms.agda [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+open import Cubical.Foundations.Univalence
+open import Cubical.Foundations.Path
+open import Cubical.Foundations.SIP
+open import Cubical.Data.Sigma
+
+private
+  variable
+    ℓ ℓ₁ ℓ₁' ℓ₂ : Level
+
+-- Now, we want to add axioms (i.e. propositions) to our Structure S that don't affect the ι.
```
Comment (mortberg) [inline Cubical/Structures/Axioms.agda:17]: Can you update this comment to something suitable? Maybe move it to the header of the file as well

**Pull request merged**